### PR TITLE
OPENEUROPA-721: Update code-review and grump.yml.dist file.

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```
Comment (pfrenssen) [inline composer.json]: There is no need to update this to 0.3 if we are not directly using the new features that are added in that version. The client still works fine with version 0.2.

I suggest to change this to `~0.2` instead, then we are compatible both with 0.2 and 0.3, and even with any future version of the code review tool.

**Comment by pfrenssen** [comment]
This is probably failing because it is now using PHPUnit 6 and the tests were written for PHPUnit 5.

File: tests/src/fixtures/parsers/components/target.yml [hunk 2]
```diff
@@ -26,7 +26,6 @@
     getAcceptedDelay : "10/07/2007"
     getAcceptedDelayFormat : "dd/mm/YYYY"
     getTranslatedFile : "File"
-    getRemark : "Remark"
```
Comment (pfrenssen) [inline tests/src/fixtures/parsers/components/target.yml:29]: Why has this been changed?

Comment (imanoleguskiza) [reply tests/src/fixtures/parsers/components/target.yml:29]: The key is duplicated in the array and Phpunit is strict about it by default.

File: composer.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
-        "phpunit/phpunit": "^5||^6.0",
+        "openeuropa/code-review": "~0.2",
+        "phpunit/phpunit": "^5||^6",
```
Comment (pfrenssen) [inline composer.json:21]: This notation with the two pipes is different than what is typically used in composer files. Normally a single pipe is sufficient. But it appears to work so I'm not rejecting it.

**Review by pfrenssen — APPROVED** [review]
OK all my questions have been addressed, looking good to me!

**Pull request merged**